Cargo.toml

```diff
@@ -13,8 +13,8 @@ crate-type = ["cdylib"]
 napi = {version="2.15", features=["napi9", "tokio_rt"]}
 napi-derive = "2.14"
 uuid = {version="1.6.1", features = ["v4","fast-rng"]}
-web-audio-api = "0.42"
-# web-audio-api = { path = "../web-audio-api-rs" }
+# web-audio-api = "0.42"
+web-audio-api = { path = "../web-audio-api-rs" }
```

Comment:
Need to wait for upstream release before merging

Reply:
Yup, will release later today